app/models/miq_ae_instance.rb

```diff
@@ -152,6 +156,11 @@ def self.get_homonymic_across_domains(user, fqname, enabled = nil, prefix: true)
     MiqAeDatastore.get_sorted_matching_objects(user, ::MiqAeInstance, ns, klass, instance, enabled)
   end
 
+  def set_fqname
```

Comment:
need a test for this

and like others, lets make sure we don't change this unless the rel_path has changed